scripts/update_schemas.py

```diff
@@ -11,6 +11,102 @@
 
 VERSION = "DSP8010_2021.2"
 
+# To use a new schema, add to list and rerun tool
```

Comment:
We removed this downstream. Don't need to do anything with this file

Reply:
Done

Reply:
> Please fix CI here as well

Done